ultradns/resource_ultradns_probe_ping.go

```diff
@@ -157,6 +162,10 @@ func makePingProbeResource(d *schema.ResourceData) (probeResource, error) {
 	p.Zone = d.Get("zone").(string)
 	p.Name = d.Get("name").(string)
 	p.ID = d.Id()
+	if len((strings.Split(string(d.Id()), ":"))) > 2 {
```

Comment:
It is set twice because the ID parameter needs to be generated in a format that we could later use it to import it back and plus the second time it is set to send actual ID in the URL while creating the resource at API end